src/WebUI/Utils.ts

```diff
@@ -122,10 +119,11 @@ export class Utils {
 
         const keys = Object.keys(imageCountMap);
         if (keys.length > 1) {

```

Comment:
same comment to rajendra also

why so much complexity, can not we simply say if container length is > 1 then do [0].image and .length, if it is > 0 then return [0] otherwise empty string.